Makefile

```
@@ -2,6 +2,11 @@ DOCKER_IMAGE_NAME=test_db_image
 DOCKER_CONTAINER_NAME=test_db_container
 DB_PORT=5432
 
+# Might be worth using poetry here so you can install packages in the venv and then
+# set the shell.
+setup-venv:
+	python -m venv .venv
```

Is there a reason to use venv over virtualenv?

```suggestion
	python -m virtualenv venv
```

---

venv is pretty standard and gnosis_chain_solver_payouts also uses venv so I think it makes sense to use `venv` for consistency.